deploy-manage/uninstall/delete-a-cloud-deployment.md

```diff
@@ -90,3 +90,31 @@ To restore a terminated deployment,
     Narrow the list by name, ID, or choose from several other filters. To further define the list, use a combination of filters.
 3. In the **Deployment Management** section, select **Restore** and then acknowledge the confirmation message.
 
+## {{eck}} [elastic-cloud-kubernetes]
+
+To delete a deployment managed by {{eck}}, you need to remove the corresponding custom resource from your Kubernetes cluster. This action deletes all {{es}} and {{kib}} pods and their associated persistent data.
+
+To delete an Elasticsearch deployment created with {{eck}}:
+
+1. Run the following command to delete the {{es}} custom resource:
+
+   ```shell
+   kubectl delete elasticsearch <deployment-name>
```

Comment:
```suggestion
   kubectl delete elasticsearch <elasticsearch-resource-name>
```

Reply:
To align with kibana example below. In ECK there's no a `deployment` concept, so we can talk about deployments in general but now when referring to a specific command.